doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -171,23 +171,87 @@ is typically done in the REST interface)
-    the same representation rules that applies to properties which represent attributes (except that a metadata
-    cannot have metadata properties).
+* The entity id is specified by the object's `id` property, which value will be a string containing the entity id.
 
-### Special attribute types
+* The entity type is specified by the object's `type` property, which value will be a string containing the entity's type name.
+
+* Entity attributes are specified by additional properties, which name is the `name` of the attribute and which representation
+  is described in the JSON Attribute Representation section below. Obviously, `id` and `type` are not allowed as attribute names.
+
+An example of this syntax in shown below:
+
+
+```
+{
+  "id": "entityID",
+  "type": "entityType",
+  "attr_1": <val_1>,
+  "attr_2": <val_2>,
+  ...
+  "attr_N": <val_N>
+}
+```
+
+
+## JSON Attribute Representation
+
+An attribute is represented by a JSON object with the following syntax:
+
+* The attribute value is specified by the `value` property, which value can be any JSON datatype.
+
+* The attribute NGSI type is specified by the `type` property, which value will be a string containing the attribute NGSI type.
+
+* The attribute metadata is specified by the `metadata` property. Its value is another JSON object which will contain a
+  property per metadata element defined (the name of the property is the `name` of the metadata element). Each metadata element,
+  in turn, is represented by a JSON object containing the following properties:
+
+  * `value`: Its value will contain the metadata value, which can correspond to any JSON datatype.
+
+  * `type`: Its value will contain a string representation of the metadata type name.
+
+An example of this syntax in shown below:
+
+```
+{
+  "value": <...>,
+  "type": <...>,
+  "metadata": <...>
+}
+```
+
+## Simplified Entity Representation
+
+There are two representation modes, which must be supported by implementations, and which allow to generate simplified representations of entities.
+
+* keyValues mode. This mode represents each entity attribute only by its value, leaving out the information about type and metadata.
+  Thus, when this mode is on, only the attribute value is represented and will correspond to the value of the property which represents the attribute.
+  See example below (To be provided).
+
+* values mode. This mode represents the entity only by a collection of attribute values conveyed by an Array. Id and type information are left out.
+  See example below (To be provided).
+
+## Partial representations
+
+Some operations imply that only a partial representation of an entity will be provided, which are summarized below:
+
+* `id` and `type` are forbidden in update operations, as they are inmutable properties.
+
+* When not forbidden, `type` can be omitted in requests. Abstent `type` means that the type is `null`.
```

Comment:
This sentence aimst to be generic to all cases, not only creation request. E.g. a /v2/rpc/query also may ommit type. Thus, I think that "Abstent `type` means that the type is `null`" suficcess all cases, being the particular interpretation of type=null operation-specific (i.e. in the case of create, it means to create an entity of type null, in the case of query it may mean that the type is not important and any type should match).